packages/go/cypher/analyzer/analyzer.go

```diff
@@ -109,14 +133,17 @@ func (s *ComplexityMeasure) onPatternPart(node *model.PatternPart) {
 		// Rendering all shortest paths could result in a large search
 		s.Weight += Weight2
 	}
+
+	return nil
 }
 
-func (s *ComplexityMeasure) onSortItem(node *model.SortItem) {
+func (s *ComplexityMeasure) onSortItem(stack *model.WalkStack, node *model.SortItem) error {
 	// Sorting incurs a weight since it will change how the projection is materialized
 	s.Weight += Weight1
+	return nil
 }
 
-func (s *ComplexityMeasure) onProjection(node *model.Projection) {
+func (s *ComplexityMeasure) onProjection(stack *model.WalkStack, node *model.Projection) error {
```

Comment:
I can't remember if it's a compiler warning or not but you may want to consider renaming "stack" to "_" since that param is unused.

Reply:
Patched up